src/main/flight/mixer.c

```diff
@@ -976,3 +970,26 @@ void filterServos(void)
 #endif
 }
 
+uint16_t pwmToMotor(uint16_t pwm)
+{
+    uint16_t motor = pwm;
+#ifdef USE_DSHOT
+    if (masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT150 || masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT600) {
+        motor = pwm == 1000 ? 0 : constrain((pwm - 977) * 2, 48, 2047);
```

Comment:
Too strict check on exactly 1000. 
If this is changed to "motor = pwm <= 1000 ? ...." it will solve the 1001 minimum bar value. 

Frankly I do not like those hard coded constants, as all/many/afew are really configurable items like min:_command etc. But I do not have any good/better/less ugly solution either. Like "977", what does that refer to ?